main.js

```diff
@@ -0,0 +1,193 @@
+// ----------BURGER MENU----------------- //
+const navbar = document.querySelector('#nav-bar');
+const burger = document.querySelector('#hamb');
+const menuLinks = document.querySelectorAll('.nav_links a');
+
+let isMenuVisible = false;
+
+function toggleMenu() {
+  if (!isMenuVisible) {
+    navbar.classList.remove('navbar');
+    navbar.classList.add('nav-bar-active');
+    burger.classList.remove('hamb');
+    burger.classList.add('nav-bar_menu-active');
+
+    // Update menu state
+    isMenuVisible = true;
+  } else {
+    // eslint-disable-next-line no-use-before-define
+    closeMenu();
+
+    // Update menu state
+    isMenuVisible = false;
+  }
+}
+
+function closeMenu() {
+  navbar.classList.remove('nav-bar-active');
+  navbar.classList.add('navbar');
+  burger.classList.remove('nav-bar_menu-active');
+  burger.classList.add('nav-bar_menu');
+
+  // Update menu state
+  isMenuVisible = false;
+}
+
+burger.addEventListener('click', toggleMenu);
+
+menuLinks.forEach((link) => {
+  link.addEventListener('click', () => {
+    if (isMenuVisible) {
+      closeMenu();
+    }
+  });
+});
+
+// ----------LINEUP SHOW MORE BUTTON----------------- //
+let areAllCardsVisible = false;
+const initialVisibleCards = 3;
+
+// Datos de ejemplo.
+const artists = [
+  {
+    name: 'Alberto Ezkivel',
+    subtitle: 'Jazz & R&B',
+    info: 'Pianist, producer and composer, co-founder of The Secret Rythm Society',
+    image: './assets/img/ezkivel.PNG',
+  },
+  {
+    name: 'Joe Kay',
+    subtitle: 'R&B, Future Beats & Hip-Hop',
+    info: 'Soulection CEO, Executive and Music Producer, DJ',
+    image: './assets/img/joekay.PNG',
+  },
+  {
+    name: 'Argo',
+    subtitle: 'R&B, Future Beats, Hip-Hop & House',
+    info: 'Executive and Music Producer, Finite Founder and CEO, DJ',
+    image: './assets/img/argo.PNG',
+  },
```

Comment:
- [ ] Hi, well done in implementing the generated dynamic section requested by the previous review. I would suggest you follow the original design. The requirements say "All page elements have similar shadows to the original design".
- **See your page**(Please pay attention to the red sign on the pictures to understand what I am talking about)
![Capture](https://github.com/rudicarrilloypr/capstone-module-1/assets/63972670/83a2c991-dca3-44fe-a6b9-c85893f156a8)
- See the original page

![Capture](https://github.com/rudicarrilloypr/capstone-module-1/assets/63972670/2d3839c1-8b14-49e0-935b-950d926be21b)